index.html

```diff
@@ -4,7 +4,182 @@
+        </span>
+        <h1 class="goal">
+          Practicing the better exercice of the heart by reaching down and
+          lifting people up.
+        </h1>
+        <div class="ideal">
+          <span
+            >Love is not what we say, love is what we do. As a song is not a
+            song until we sing it. A bell is not a bell until we ring it. In the
+            same way <strong>Love</strong> is not love until we show it
+            away</span
+          >
+          <h3 class="work-hour">Everyday 8am - 5pm(MON) ~ (FRI) 2pm</h3>
+          @ Genuine charity, from all over the places
+        </div>
+      </section>
+      <section class="program" id="program">
+        <h3 class="program-title">Main Program</h3>
+        <div class="program-container">
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/motivation.png" alt="motivation-icone" />
+              <span>Motivation</span>
+            </div>
+            <div class="right">
+              Listen to the motivation form various personalities about the
+              message of sharing and opening opportunities to the less
+              privileged in your community.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/meeting.png" alt="forum-icone" />
+              <span>Forum</span>
+            </div>
+            <div class="right">
+              Have the time to share your toughts and opinions with our team for
+              each topic.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/donation.png" alt="donation-icone" />
+              <span>Donation</span>
+            </div>
+            <div class="right">
+              The value of life is not in its duration, but in its donation.
+              What a privilege to be here on the planet to contribute your
+              unique donation to humankind!
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/support.png" alt="support-icone" />
+              <span>Support</span>
+            </div>
+            <div class="right">
+              Whatever little it maybe, it means a lot to someone out there.
+              Support the needy in our community.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/workshop.png" alt="workshop-icone" />
+              <span>Workshop</span>
+            </div>
+            <div class="right">
+              Get oppotunities to network with GC affiliates around the globe.
+            </div>
+          </div>
+        </div>
+        <div class="join-btn">
+          <a href="#">Join Genuine Charity</a>
+        </div>
```

Comment:
I hope as long as we had to personalize the template on our own. this element here being like this Is great fora better user experience and even the marketing of the organization I have worked on. The difference is that for the provided template it was only to see the program while in my case It is a request to the user to join our program. so I wanted to make sure that is really noticeable @brytebee 
Could even add some animation on it!.. so that it may be very visibible

Reply:
not resolved

Reply:
let me fix that real quick.
I was rushing that why I did not change that.
thank you again for the review :pray: